SpatialGDK/Source/SpatialGDK/Private/Tests/SpatialView/DispatcherTest.cpp

```diff
@@ -0,0 +1,136 @@
+﻿// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include <memory>
+
+#include "ComponentTestUtils.h"
+#include "SpatialView/Callbacks.h"
+#include "SpatialView/ComponentData.h"
+#include "SpatialView/Dispatcher.h"
+#include "SpatialView/EntityDelta.h"
+#include "SpatialView/EntityView.h"
+#include "SpatialView/OpList/EntityComponentOpList.h"
+#include "SpatialView/ViewDelta.h"
+#include "SpatialViewUtils.h"
+#include "Tests/TestDefinitions.h"
+
+#define DISPATCHER_TEST(TestName) GDK_TEST(Core, Dispatcher, TestName)
+
+namespace
+{
+constexpr Worker_ComponentId COMPONENT_ID = 1000;
+constexpr Worker_ComponentId OTHER_COMPONENT_ID = 1001;
+constexpr Worker_EntityId ENTITY_ID = 1;
+constexpr Worker_EntityId OTHER_ENTITY_ID = 2;
+constexpr double COMPONENT_VALUE = 3;
+constexpr double OTHER_COMPONENT_VALUE = 4;
+
+void PopulateViewDeltaWithComponentAdded(SpatialGDK::ViewDelta& Delta, SpatialGDK::EntityView& View, const Worker_EntityId EntityId,
+										 const Worker_ComponentId ComponentId, const double Value)
+{
+	SpatialGDK::EntityComponentOpListBuilder OpListBuilder = SpatialGDK::EntityComponentOpListBuilder{};
```

Comment:
Nit - don't need the equals or stuff after it. This has a default constructor.
General style in spatial view has been to use universal initialiser stuff only for pod or stuff that looks like pod.